components/automate-ui/README.md

```diff
@@ -55,24 +55,17 @@ Certain packages in package.json are constrained for the reasons detailed here.
 At any future moment though, the reasons for constraint here could be invalidated, so this should be updated as needed when package.json is updated.
 While it is problematic to document this information due to the maintenance burden, the value of having this in one place outweighs that burden.
 
```

Comment:
I suspect this package just bumped from 165 to 166 in the midst of your work and could be updated to remove it from our exceptions list.
```
@types/lodash         4.14.165  4.14.166     4.14.166
```

Reply:
Thanks for finding this!